reaktive/src/commonMain/kotlin/com/badoo/reaktive/plugin/SimpleReaktivePlugin.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.reaktive.plugin
+
+import com.badoo.reaktive.annotations.ExperimentalReaktiveApi
+import com.badoo.reaktive.observable.Observable
+
+@ExperimentalReaktiveApi
+open class SimpleReaktivePlugin : ReaktivePlugin {
```

Comment:
What is the point of this class?
Maybe better to provide default implementation of each method in `ReaktivePlugin`?

Reply:
For some reason I don't like methods with implementations in interfaces. They may cause confusion/conflicts/bugs with multiple/nested inheritance. The purpose of this class is you can extend it and implement required methods. Do you think in this particular case it would be good to add default implementations in interface?

Reply:
You can keep it as is if you want.